src/v/config/configuration.cc

```diff
@@ -660,6 +660,41 @@ configuration::configuration()
         model::fetch_read_strategy::non_polling,
         model::fetch_read_strategy::non_polling_with_debounce,
       })
+  , fetch_pid_p_coeff(
+      *this,
+      "fetch_pid_p_coeff",
+      "proportional coefficient for fetch PID controller.",
+      {.needs_restart = needs_restart::no, .visibility = visibility::tunable},
+      10,
+      {.min = 1, .max = std::numeric_limits<int64_t>::max()})
+  , fetch_pid_i_coeff(
+      *this,
+      "fetch_pid_i_coeff",
+      "integral coefficient for fetch PID controller.",
+      {.needs_restart = needs_restart::no, .visibility = visibility::tunable},
+      100000,
+      {.min = 1, .max = std::numeric_limits<int64_t>::max()})
+  , fetch_pid_d_coeff(
+      *this,
+      "fetch_pid_d_coeff",
+      "derivative coefficient for fetch PID controller.",
+      {.needs_restart = needs_restart::no, .visibility = visibility::tunable},
+      0,
+      {.min = 0, .max = std::numeric_limits<int64_t>::max()})
+  , fetch_pid_target_fetch_sg(
+      *this,
+      "fetch_pid_target_fetch_sg",
+      "The target resource utilization of the fetch scheduling group between 1 "
+      "and 10,000",
+      {.needs_restart = needs_restart::no, .visibility = visibility::tunable},
+      2000,
```

Comment:
I guess that it's technically unit-less. Let me change it to be a percentage between 0.0 and 1.0 and then convert to the int representation internally.

Reply:
Changed this property to be a percentage between 0.0 and 1.0. Hopefully that'll be more intuitive.

Reply:
@ballard26 see above.

Reply:
I was just representing the percentage as a float `a` where `100 * a` would be the integer percentage. Can switch the property to an integer in `(0, 100]` if that makes more sense to folks.

Reply:
"percent" (i.e., _per one hundred_)is always based on a maximum of 100, even it's a float: then 100.0 would the maximum. If you want to use a 0-1 scale it could be "fraction of ..." not "percent". I recommend percent though, fine to keep it a float which I guess is useful test really low values.

Reply:
Also a good idea to include the unit as the property suffix.